### ISSUE TITLE
Add Volume Controller

### ISSUE THREAD

**Issue by Siddharth-Dagar-25** [Siddharth-Dagar-25/Geet-Music-Player#5]
# Feature Request: Add Volume Controller

## Objective :
Incorporate a volume control feature into the Track.js component, enabling straightforward adjustment of the song's volume level.

### Development and Contribution Process

- Fork the repository to begin working on this feature.
- Maintain the integrity of existing code:
 -- Do not alter existing variable names, indexes, or IDs.
 -- Utilize the existing data structures and variables wherever possible.
- Follow best practices for naming new variables and functions to ensure code clarity and maintainability.
- Thoroughly comment your code to explain the logic and functionality of the added features.
- Ensure that the new features do not negatively impact the existing functionalities of the music player.

### Final Steps

- Once development is complete, test the new features to verify they work as expected without introducing bugs or affecting the existing functionality.
- Submit your changes via a Pull Request (PR) to the original repository for review.

Your contribution will enhance the music listening experience by providing users with greater control over their playback options and ensuring uninterrupted enjoyment of their favorite songs.

**Comment by Sky-De** [comment]
@Siddharth-Dagar-25 i would like to fix this, please assign this to me

**Comment by Siddharth-Dagar-25** [comment]
> @Siddharth-Dagar-25 i would like to fix this, please assign this to me

Thank you for your interest but it is already done. You can look out for other issues.
